lib/schemas/global/apiCourtCentre.json

```diff
@@ -24,8 +27,20 @@
     "address": {
       "$ref": "http://justice.gov.uk/core/courts/external/apiAddress.json"
     },
-	"psaCode": {
+    "psaCode": {
       "type": "integer"
+    },
+    "welshAddress": {
+      "$ref": "http://justice.gov.uk/core/courts/external/apiAddress.json",
+      "description": "The court address in welsh language.  Required for generation of welsh documentation.  Required to be enriched when publishing hearing results"
+    },
+    "welshCourtCentre": {
+      "description": "Indicates that the court centre is located under Welsh regulation i.e. is in Wales",
+      "type": "boolean"
+    },
+    "lja": {
+      "description": "The lja details of the court centre.  Required for generation of documentation.  Required to be enriched when publishing hearing results",
+      "$ref": "http://justice.gov.uk/core/courts/external/apiLjaDetails.json"

```

Comment:
Changes here have me expecting a model and factory change and a migration for courtCentre which I can't find. 

We've added:
code
welshAddress
welshCourtCentre
lja

Reply:
CourtCentre is actually extracted out into a gem: https://github.com/ministryofjustice/hmcts_common_platform

Reply:
So there is still a CourtCentre model that uses values received directly (not looking them up via the gem( https://github.com/ministryofjustice/hmcts-common-platform-mock-api/blob/master/app/models/court_centre.rb

We also need to revisit the gem and consider if we still need it given what data we are now getting sent